kolibri/plugins/learn/assets/src/views/TopicsPage/ToggleHeaderTabs.vue

```diff
@@ -90,7 +90,6 @@
         // navigates the main page to the search view
         if (this.topic) {
           const query = { ...this.$route.query };
-          delete query.dropdown;
```

Comment:
It was because we used to have a query parameter that determined whether the side panel state was toggled open, I think? Then we refactored it to rely on component JS state, but this was never cleaned up.